src/core/include/openvino/op/util/topk_base.hpp

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include "openvino/op/constant.hpp"
```

Comment:
```suggestion
```

Reply:
let us include this where it is really needed

Reply:
Is added berceuse is required.

Reply:
I can't see that this is required in `topk_base.hpp`. Just include this in `.cpp` where it is used.

Reply:
it is used ad line 96:
` size_t validate_and_get_k(const std::shared_ptr<op::v0::Constant>& k_constant) const;`

Reply:
If critical, let's use forward declaration

Reply:
Changed to forward declaration